templates/planning/index.html.twig

```diff
@@ -5,12 +5,12 @@
 {% block content %}
     <h1 class="page-title">{{ 'planning.title'|trans }}</h1>
 
-    <div class="selections">
         {{ form_start(form) }}
         {{ form_row(form.dataProvider) }}
         {{ form_row(form.viewType) }}
-        <button class="button m-5">{{ 'planning.search'|trans }}</button>
+        <div>
```

Comment:
What is the div used for?

Reply:
To give the button an anchor point in relation to the other form elements, which are wrapped in divs. Now, they are in line. Before, the button were never in line.